### PR TITLE
Post edit and delete

### PR DESCRIPTION
**Pull request by HeyNitin**
Added edit and delete functionality on post page

https://deploy-preview-8--strivesocial.netlify.app/

### CODE REVIEW

**Comment by shivamSspirit** [comment]
land page is collapse

File: src/pages/postPage/postPage.tsx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
 import { useAppDispatch, useAppSelector } from "appRedux/hooks"
 import { commentedBy, postTypes, setComments, setPosts } from "appRedux/postSlice"
 import axios from "axios"
+import { AddPosts } from "components/addPosts/addPosts"
```
Comment (parulgupta2001) [inline src/pages/postPage/postPage.tsx]: import all components from one file i.e index.js

**Review by parulgupta2001 — APPROVED** [review]
Only one suggestion otherwise everything looks good

**Comment by HeyNitin** [comment]
> land page is collapse

What do you mean by collapse? I didn't understand

**Pull request merged**